FxSsh/Services/UserauthService.cs

```diff
@@ -35,33 +36,76 @@ private void HandleMessage(RequestMessage message)
             switch (message.MethodName)
             {
                 case "publickey":
-                    var msg = Message.LoadFrom<PublicKeyRequestMessage>(message);
-                    HandleMessage(msg);
+                    var keyMsg = Message.LoadFrom<PublicKeyRequestMessage>(message);
+                    HandleMessage(keyMsg);       
```

Comment:
trim the white space at end
```suggestion
                    HandleMessage(keyMsg);
```